server/task_executor_schema_update_ghost_sync.go

```diff
@@ -162,11 +162,14 @@ func runGhostMigration(ctx context.Context, server *Server, task *api.Task, stat
 
 	waitSync := &sync.WaitGroup{}
 	waitSync.Add(1)
+	syncDone := make(chan struct{})
```

Comment:
The current implementation is complicated.
If we want to deal with
1. waiting for gh-ost migration cutover
2. return on error
at the same time, maybe it's better to only use channels and re-design.

Reply:
Sure, `syncDone` alone should serve the purpose well. I'll remove `waitSync`.